### DISCUSSION TITLE
dot selection vs pipe operator

### DISCUSSION THREAD

**Discussion by xialvjun** [koka-lang/koka#337] [General]
`s.encode` vs `s |> encode`

but what if `s |> some_module.encode`

**Comment by axhixh** [comment]
What is being discussed here? `|>` is familiar from other languages. Certainly, what I am used to. However, I find `.` equally nice. In fact, it looks more familiar to most programmers. I like that you can do `s.some_module/encode` I would have written this as `s.some-module/encode` or imported `some-module` so i could write `s.encode`.

The main thing is that, familiarity with other languages is restricting us from seeing what Koka is offering.

**Comment by woojamon** [comment]
As an F# developer that frequently uses `|>`, I actually find that I like the `.` style much better.
